action_blockheader.go

```diff
@@ -44,3 +44,30 @@ func (c *Client) RecordBlockHeader(ctx context.Context, hash string, bh bc.Block
 	// Return the response
 	return blockHeader, nil
 }
+
+// GetUnsyncedBlockHeaders get all unsynced block headers
+func (c *Client) GetUnsyncedBlockHeaders(ctx context.Context) ([]*BlockHeader, error) {
+
+	// Check for existing NewRelic transaction
+	ctx = c.GetOrStartTxn(ctx, "get_unsynced_blockheaders")
+
+	return GetUnsyncedBlockHeaders(ctx, c.DefaultModelOptions()...)
```

Comment:
Why is this a public function?

Reply:
All actions are public functions, right?